posts/TechYesWeCan_Pt1.md

```diff
@@ -0,0 +1,115 @@
+
+**Kate Foster | Principal | KickStart Specialists**
+
+I have said in the past that getting a degree in Business Information Systems made a world of difference in my career. Having that engineering background with a mix of business concepts and understanding has helped me maintain relationships with business leaders within every project I have been a part of. Hearing Kate Foster speak towards just this was perfect!
+
+IT and business people don’t speak the same language, everyone knows that. But what few engineers learn early on is that you must be able to translate for the business. IT “gobbledygook” causes problems -- not matter how good your solution is. 
+
+What good are innovative ideals if you can’t explain why they’re good. Ineffective communication is one of the primary contributions to project failures ⅓ of the time (and has a negative effect ½ of the time). 
+And even if you know how to communicate, how do you know the message you’re sending is what they’re receiving? Get to know the client! And get to know yourself. 
+
+**Avis Yates Rivers | CEO | Technology Concepts Group International**
+Women hold a meaningful percentage of participation in the technology world. We comprise 57% of US professional occupations, 26% of US tech jobs and 19% of the US software developers. These numbers sound about right! We are growing in this industry and I feel a shift in the balances. But even with all that change and time we are still only hold 5% of technology leadership jobs. Avis Rivers spoke to this idea; that the gaps in this industry aren’t just about pay but also still with gender and race.
+
+Being aware of the problem, she described best as intersectionality, we can better acknowledge and ground the differences among us. We can break out of these discriminations and disadvantages. 
+
+Step 1: “Bring your whole authentic self to work and what your doing”. We’re all different. Diversity of thought leads to better problem solving. But it’s this combination that drives innovation! Engage all the voices on your team. Don’t filter questions or solutions based on race or age. We need everyone represented in each business. Change the dynamic!
+
+Step 2: Understand that equality doesn’t work for everyone. Equity is what we should be aiming for to create success! Rivers showed an image, that many people probably recognize, during her talk. It resonated with me and really shows the difference in these two types of strategies vs the reality of life.
+
+<responsive-figure src="https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/10/Leigha_IMG1.png">
```

Comment:
Use the `~[title](url)` syntax

Reply:
Um....I think I did this - I may need help!